pre_commit_hooks/check_case_conflict.py

```diff
@@ -12,9 +13,23 @@ def lower_set(iterable: Iterable[str]) -> Set[str]:
     return {x.lower() for x in iterable}
 
 
+def parents(file: str) -> Iterator[str]:
+    while True:
+        file = file[:file.rfind('/')]
```

Comment:
I believe you can use `os.path.dirname` for this

Reply:
done, it seems to work